data_model/primitives/src/conststr.rs

```diff
@@ -0,0 +1,670 @@
+//! Const-string related implementation and structs.
+extern crate alloc;
```

Comment:
I don't think you need this, it's inclded in the `data_model/lib.rs`

Reply:
This is supposed to be a separate crate, so it might not be. @Erigara please check

Reply:
@mversic, I get errors when deleting this line.

Reply:
You shouldn't be getting any errors. At least add `#[cfg(not(feature = "std"))]`

If this'll be a separate crate there should be no `std` feature in it